users/config.rst

```diff
@@ -354,9 +354,12 @@ copyOwnershipFromParent
     Requires running Syncthing as privileged user, or granting it additional capabilities (e.g. CAP_CHOWN on Linux).
 
 modTimeWindowS
-    Allowed modification timestamp difference when comparing files for equivalence.
-    To be used on systems that have unstable modification timestamps, that might change after being observed after
-    the last write operation. Used in Android only.
+    Allowed modification timestamp difference when comparing files for
+    equivalence. To be used on file systems which have unstable or
+    imprecise modification timestamps that might change after being
```

Comment:
Imprecise timestamps are not a problem, only unstable ones. For an attempted explanation see: https://forum.syncthing.net/t/why-default-modtimewindows-to-2-for-fat-only-on-android/16256/2